interface/settings.py

```diff
@@ -108,6 +108,7 @@
 MINIO_BUCKET = os.environ.get('MINIO_BUCKET', 'test')
 
 ACS_INTERFACE_ADDRESS = os.environ.get('ACS_INTERFACE_ADDRESS', 'localhost:8100')  # noqa: E501
+ACS_USER_WHITELIST = os.getenv('ACS_USER_WHITELIST', ',').split(',')
```

Comment:
`','` makes no sense as a default, and even `''` will result in the split value `['']`. The clean way is to use an intermediate variable:

```python
_acs_user_whitelist = os.getenv('ACS_USER_WHITELIST', None)
ACS_USER_WHITELIST = _acs_user_whitelist.split(',') if _acs_user_whitelist else []